src/com/spikes2212/motor/Gearbox.java

```diff
@@ -7,59 +7,60 @@
 
 import java.util.List;
 
-public class Gearbox implements SpeedController {
+public class Gearbox extends WPI_TalonSRX{
 
-    private WPI_TalonSRX master;
     private List<SpeedController> victorSPs;
+    private int port;
 
-    public Gearbox(WPI_TalonSRX master) {
-        this.master = master;
+    public Gearbox(int port) {
+        super(port);
+        this.port = port;
     }
 
     @Override
     public void set(double speed) {
```

Comment:
there is no need to override `WPI_TalonSRX` methods. they are already implemented inside the super-class :)